test/test_sanity.py

```diff
@@ -232,14 +232,14 @@ def test_resfinder():
         assert result.reference_database_id == 'resfinder'
         assert result.reference_database_version == '2019-Jul-28'
         assert result.reference_accession == 'EU370913'
-        assert result.analysis_software_name == 'resfinder.py'  # drop the .py?
+        assert result.analysis_software_name == 'resfinder 3'  # drop the .py?
```

Comment:
Reasonable, I had .py as at certain points I've found the CLI resfinder and web portal have diverged a bit, but this is a good change.